.github/workflows/release_asset.yml

```
@@ -0,0 +1,51 @@
+name: Release asset bot
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+jobs:
+  build:
+    name: Build release asset
+    continue-on-error: true
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - ubuntu-latest
+          - ubuntu-18.04
+          - macos-latest
+          - windows-latest
+    runs-on: ${{matrix.platform}}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: install rust toolchain
+        uses: actions-rs/toolchain@v1
+      - name: build for ${{ matrix.platform }}
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --release
+
+      - name: build move release asset
+        run: bash ./scripts/release.sh ${{ matrix.platform }}
```

You shouldn't need to specify `bash` here
```suggestion
        run: ./scripts/release.sh ${{ matrix.platform }}
```

---

Got it, I'll remove it

---

On the windows platform, there is a problem with removing bash, I added it back through this PR